ExpandRegion.py

```diff
@@ -81,18 +82,25 @@ def run(self, edit, language="", undo=False, debug=True):
         if debug:
           print("startIndex: {0}, endIndex: {1}, type: {2}".format(result["start"], result["end"], result["type"]))
       else:
-        # if there is no result, keep the current region
-        new_regions.append(region)
-
-    # replace the selections with the new regions
-    view.sel().clear()
-    for sel in new_regions:
-      view.sel().add(sel)
-
-    settings = sublime.load_settings("ExpandRegion.sublime-settings")
-    do_force_enable_soft_undo = settings.get("force_soft_undo_integration")
-    if do_force_enable_soft_undo:
-      _force_enable_soft_undo(view, edit, new_regions)
+        # # if there is no result, keep the current region
```

Comment:
Remove double `#`

Reply:
Also: does the comment still make sense here?

Reply:
Not much, I changed the comment to incorporate new changes.